.github/workflows/publish-all.yml

```diff
@@ -25,10 +25,10 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v3
     
-    - name: Setup .NET 6
-      uses: actions/setup-dotnet@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: 6.0.x
+        dotnet-version: 3.1.x
```

Comment:
Reverting back to .net core 3.1.x as dotnet nuget publish --skip-duplicates doesn't work with .net 6 for some reason :D

Reply:
Does it mean that we'll be stuck on 3.1? 😄 I tried googling very quickly and found issues like [this](https://github.com/microsoft/azure-pipelines-tasks/issues/13591). Would [the solution](https://github.com/microsoft/azure-pipelines-tasks/issues/13591#issuecomment-793933011) with `allowPackageConflicts` help in this case?

Reply:
_Does it mean that we'll be stuck on 3.1?_ Not really, I will need to figure out why `--skip-duplicates` doesn't work as it should.

I will revisit this sometime later, i have it on my todo list :) but for now, we can keep the publish pipelines to use .net core 3.1, that shouldn't be an issue for now.